builtin/logical/pki/backend_test.go

```diff
@@ -3549,6 +3549,85 @@ func TestBackend_URI_SANs(t *testing.T) {
 	}
 }
 
+func TestBackend_IP_SANs(t *testing.T) {
+	t.Parallel()
+	b, s := CreateBackendWithStorage(t)
+
+	var err error
+
+	_, err = CBWrite(b, s, "root/generate/internal", map[string]interface{}{
+		"ttl":         "40h",
+		"common_name": "example.com",
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
If you want, there's a few useful helpers like `requireSuccessNonNilResponse(...)` and you can use `require.*` functions. But happy to take this as-is as well, I think it is sufficient.